src/CppUTest/JUnitTestOutput.cpp

```diff
@@ -135,6 +136,9 @@ void JUnitTestOutput::printCurrentTestStarted(const UtestShell& test)
         impl_->results_.tail_ = impl_->results_.tail_->next_;
     }
     impl_->results_.tail_->name_ = test.getName();
+    if (*(test.getProgressIndicator()) == '!') {
```

Comment:
This is kind of an ugly way of discovering ignored. Would it be better to add an willRun method or so to the uTest class?